includes/definitions/discovery/audiocodes.yaml

```diff
@@ -0,0 +1,120 @@
+mib: AC-ANALOG-MIB:AcGateway:AC-PSTN-MIB
+modules:
+    sensors:
+        pre-cache:
+            data:
+                - oid:
+                    - callerDisplayCallerDisplay
+                    - channelsStartingPhoneNum
+                    - acAnalogFxsFxoType
+                    - acTrunkDialPlanName 
+        state:
+            data:
+                -
+                    oid: acAnalogFxsFxoTable
+                    value: acAnalogFxsFxoHookState
+                    num_oid: .1.3.6.1.4.1.5003.9.10.1.4.1.1.18
```

Comment:
This OID is used in the next 3 sensors. I assume that's a mistake.

Can't see any test data in the snmprec file under 1.3.6.1.4.1.5003.9.10.1.

Reply:
It is made on purpose and works as expected, but is clearly "borderline" ;).  It is the only way I found to show the fact that some of these entries could be FXO, FXS and E&M. Each of the 3 occurrences of the OID is filtered depending the type so the entry can have a proper text. 
The way to avoid this would be to find a way to map the value of acAnalogFxsFxoType from "Integer" to the FXO, FXS and E&M. 
Any way to do this ?

Reply:
Do you see any reason this yaml file would fail ? The result is perfect on my device (FXS only) but should work as well on the 3 others, and even on products with a mix of those.

Reply:
No it looks correct